source/utils/unhandle.ts

```diff
@@ -2,7 +2,7 @@ import EventEmitter = require('events');
 
 type Origin = EventEmitter;
 type Event = string | symbol;
-type Fn = (...args: unknown[]) => void;
+type Fn = (...args: any[]) => void;
```

Comment:
This matches how it's typed upstream (from `import EventEmitter = require('events');`)